secret/vault/vault.go

```diff
@@ -37,8 +48,16 @@ func New(addr, token string) (*client, error) {
 func secretFromVault(vault *api.Secret) *library.Secret {
 	s := new(library.Secret)
 
+	var data map[string]interface{}
+	// handle k/v v2
+	if _, ok := vault.Data["data"]; ok  {
```

Comment:
Response returned via k/v v2 are underneath the `data` key - https://www.vaultproject.io/api-docs/secret/kv/kv-v2#sample-response-4.